search-app/src/cmr/search/services/humanizers/humanizer_report_service.clj

```diff
@@ -127,23 +128,14 @@
         (warn (.getMessage e) msg/returning-empty-report)
         (throw e)))))
 
-(defconfig report-cache-consistent-timeout-seconds
```

Comment:
Removed this because the humanizer report is an extremely infrequent request so we are fine with always checking cubby to make sure our cache is up to date on this node when requesting the humanizer report.